common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -1551,8 +1551,8 @@ bool ap_wlan_hal_dwpal::read_supported_channels()
         }
     }
 
-    m_radio_info.preferred_channels.insert(m_radio_info.preferred_channels.begin(),
-                                           supported_channels.begin(), supported_channels.end());
+    m_radio_info.supported_channels.insert(m_radio_info.supported_channels.begin(),
```

Comment:
This looks like a bug - isn't it?
```suggestion
    m_radio_info.supported_channels.insert(m_radio_info.supported_channels.begin(),
                                           supported_channels.begin(), supported_channels.end());
```

Reply:
Now that I look at it again - this is part of a functino which shouldn't exist in this commit - `bool ap_wlan_hal_dwpal::read_supported_channels()` - I'm confused, didn't the previous commit replace all occurrences of supported channels with preferred channels?